code/DevCheckController.php

```diff
@@ -6,10 +6,15 @@ class DevCheckController extends Controller {
 		'index'
 	);
 
+	/**
+	 * @var string Permission code to check for access to this controller.
+	 */
+	private static $permission = 'ADMIN';
+
 	function index($request) {
 		$suiteName = $request->param('Suite') ? $request->param('Suite') : 'check';
 		$e = new EnvironmentChecker($suiteName, 'Environment status');
-		$e->init('ADMIN');  //check for admin permissions before running this check
+		$e->init($this->config()->permission);  //check for admin permissions before running this check
```

Comment:
hah.. i'm stupid. I didn't know you could use a magic property getter. I've always used $this->config->get('permission');